src/sqlite3ext.h

```diff
@@ -370,6 +370,7 @@ struct libsql_api_routines {
   struct libsql_wal_methods *(*wal_methods_find)(const char *);
   int (*wal_methods_register)(struct libsql_wal_methods*);
   int (*wal_methods_unregister)(struct libsql_wal_methods*);
+  void *(*close_hook)(sqlite3*, void(*)(void*,sqlite3*), void *pArg);
```

Comment:
that one will be available in libSQL 0.2.3 - let's add a comment, following SQLite style above